src/pages/ProblemEditor/ProblemEditorContainer/problemEditorContainerSlice.spec.ts

```diff
@@ -0,0 +1,288 @@
+import problemEditorContainerSlice, {
```

Comment:
Question: was this copy pasted from the past src folder or are these new tests?

Reply:
i meant to delete it and commit it, will do it now.